src/data/describeIndexStats.ts

```diff
@@ -60,16 +53,23 @@ export type DescribeIndexStatsOptions = {
 };
 
 export const describeIndexStats = (apiProvider: DataOperationsProvider) => {
-  const validator = buildConfigValidator(
-    DescribeIndexStatsOptionsSchema,
-    'describeIndexStats'
-  );
+  const validator = async (options: DescribeIndexStatsOptions) => {
+    if (options) {
+      ValidateProperties(options, ['filter']);
+    }
+    const map = options['filter'];
+    for (const key in map) {
+      if (!map[key]) {
+        throw new PineconeArgumentError('Filter cannot be empty');
```

Comment:
Could maybe add the `key` value to the error message here:
```suggestion
        throw new PineconeArgumentError(`Filter cannot be empty for ${key}`);
```